Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml

```diff
@@ -2,71 +2,71 @@
   id: HumanHairAfro
   bodyPart: Hair
   markingCategory: Hair
-  speciesRestriction: [Human, Dwarf, SlimePerson, Reptilian, Moth]
+  speciesRestriction: [Human, Dwarf, SlimePerson, Reptilian, Moth, Vulpkanin] # CD: add Vulps and moths
```

Comment:
Technically reptilians should be on the header too as their hair is a CD change. Not the end of the world though since it being there at all is an improvement.

Reply:
Oh, I did not know that. That is one sed command.